src/frontend/src/features/layouts/components/thread-view/components/calendar-invite/_index.scss

```
@@ -76,14 +83,14 @@
 
 .calendar-invite__title {
     margin: 0;
-    font-size: var(--c--globals--font--sizes--lg);
+    font-size: var(--c--globals--font--sizes--md);
     font-weight: 600;
     color: var(--c--contextuals--content--semantic--neutral--primary);
     word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)